Packages/MIES/MIES_Utilities.ipf

```diff
@@ -3156,14 +3158,28 @@ threadsafe Function/S NumericWaveToList(WAVE/Z wv, string sep, [string format])
 		format = "%g"
 	endif
 
+	if(ParamIsDefault(colSep))
+		colSep = ","
```

Comment:
Would it make sense to have some default separator constants defined globally that are used preferrentially?
If yes, lets make an issue for that.

Reply:
Good point. An issue would be good.